__tests__/utils/teamListUtils.js

```diff
@@ -0,0 +1,59 @@
+import { shouldBeScored } from '../../app/utils/teamListUtils'
+
+describe('teamListUtils shouldBeScored', () => {
+    it('should be false when there is exactly one team and we are on the first round', () => {
+        // Arrange
+        const aTeam = {
+            isInPlay: (round) => true
+        }
+        const teamList = [aTeam]
+
+        // Act
+        const result = shouldBeScored(teamList, aTeam, 0)
+
+        // Assert
+        expect(result).toBeFalsy()
+    })
+
+    it('should be false when the target team is in the first position or end of array and its the first round', () => {
+        // Arrange
+        const aTeam = {
+            isInPlay: (round) => true
+        }
+        const teamList = [{}, aTeam]
```

Comment:
So I see how it's at the end of the array but I'm a bit confused as to how this is testing it for both the first position and the end of the array. Perhaps this should be made into two tests.

Reply:
Updated the test name and added a comment we can reopen this or create an issue if you feel strongly about it!